numpy/testing/_private/utils.py

```diff
@@ -695,9 +695,14 @@ def isnumber(x):
     def istime(x):
         return x.dtype.char in "Mm"
 
-    def chk_same_position(x_id, y_id, hasval='nan'):
-        """Handling nan/inf: check that x and y have the nan/inf at the same
-        locations."""
+    def func_chk_same_pos(x, y, func=isnan, hasval='nan'):
```

Comment:
Can we spell this `check` instead of `chk`? Or even `assert_same_predicate_pos`?

Reply:
OK, `assert_same_pos`